docs/content/configuration/policy-resource.md

```diff
@@ -547,9 +547,11 @@ waf:
 |Field | Description | Type | Required |
 | ---| ---| ---| --- |
 |``enable`` | Enables NGINX App Protect WAF. | ``bool`` | Yes |
-|``apPolicy`` | The [App Protect WAF policy](/nginx-ingress-controller/app-protect/configuration/#app-protect-policies) of the WAF. Accepts an optional namespace. | ``string`` | No |
+|``apPolicy`` | The [App Protect WAF policy](/nginx-ingress-controller/installation/integrations/app-protect-waf/configuration/#nginx-app-protect-waf-policies) of the WAF. Accepts an optional namespace. Mutually exclusive with ``apBundle``. | ``string`` | No |
+|``apBundle`` | The [App Protect WAF policy bundle](/nginx-ingress-controller/installation/integrations/app-protect-waf/configuration/#nginx-app-protect-waf-bundles). Mutually exclusive with ``apPolicy``. | ``string`` | No |
 |``securityLog.enable`` | Enables security log. | ``bool`` | No |
-|``securityLog.apLogConf`` | The [App Protect WAF log conf](/nginx-ingress-controller/app-protect/configuration/#app-protect-logs) resource. Accepts an optional namespace. | ``string`` | No |
+|``securityLog.apLogConf`` | The [App Protect WAF log conf](/nginx-ingress-controller/installation/integrations/app-protect-waf/configuration/#nginx-app-protect-waf-logs) resource. Accepts an optional namespace. Only works with ``apPolicy``. | ``string`` | No |
+|``securityLog.apLogBundle`` | The [App Protect WAF log bundle](/nginx-ingress-controller/installation/integrations/app-protect-waf/configuration/#nginx-app-protect-waf-bundles) resource. Only works with ``apBundle``. | ``string`` | No |
```

Comment:
These should probably all follow the relref Hugo format: they're more robust and will fire errors if a link breaks in the future.

Reply:
Updated, thanks @ADubhlaoich!